CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ## XX.XX.XX
 * Disabled caching for webviews.
 
+* Fixed an issue where the system was incorrectly validating parameters during content retrieval.
+* Mitigated an issue by ensuring that the resize_me action now correctly accounts for display density when adjusting content size.
```

Comment:
resize_me is sth users wont know. So instead mention something about density not being considering while resizing the widgets and such